operator/pkg/central/values/translation/image_overrides.go

```diff
@@ -5,6 +5,7 @@ import "github.com/stackrox/rox/operator/pkg/images"
 var (
 	imageOverrides = images.Overrides{
 		images.Main:      "central.image.fullRef",
+		images.CentralDB: "central.db.image.fullRef",
```

Comment:
Took this from https://github.com/stackrox/stackrox/blob/4e108b1d026b17a2fc986def22b41e563c43ba78/image/templates/helm/stackrox-central/templates/01-central-12-central-db.yaml#L86